homeassistant/components/plugwise/climate.py

```diff
@@ -60,13 +65,14 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
The library doesn't support asyncio. We're not allowed to do sync I/O inside the event loop.

Please revert the change to use the async api.

Reply:
What are you saying here? Revert only the change shown above? Or all async changes?

Reply:
All async changes, please.

Reply:
Ok understood.
How to make the haanna-library asyncio compatible? Can you advise? Point to some example?

Reply:
Since haanna currently uses requests we can change to use aiohttp. Please have the library accept an optional aiohttp session so we can reuse the aiohttp session of the app, home assistant.

See eg: https://github.com/Danielhiversen/pymill

Reply:
Thanks! This need need some time I see...

Reply:
All async-related code was removed.